satellite/audit/reporter.go

```diff
@@ -181,6 +181,8 @@ func (reporter *Reporter) recordAuditSuccessStatus(ctx context.Context, successN
 func (reporter *Reporter) recordPendingAudits(ctx context.Context, pendingAudits []*PendingAudit) (failed []*PendingAudit, err error) {
 	defer mon.Task()(&ctx)(&err)
 	var errlist errs.Group
+
+	var updateRequests []*overlay.UpdateRequest
```

Comment:
shouldn't you add the length of pendingAudits > 0 check? else return nil with a warning? and at other similar usages